### ISSUE TITLE
Windows: AttributeError: 'module' object has no attribute 'SIGUSR1'

### ISSUE THREAD

**Issue by a-khakimov** [bprinty/animation#6]
```python
In [1]: import animation
```
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-1-d82fc6a65b0f> in <module>()
----> 1 import animation

c:\python27\lib\site-packages\animation\__init__.py in <module>()
      6
      7
----> 8 from .decorators import wait, simple_wait, Wait

c:\python27\lib\site-packages\animation\decorators.py in <module>()
     31 signal.signal(signal.SIGINT, end_wait_threads)
     32 signal.signal(signal.SIGINT, end_wait_threads)
---> 33 signal.signal(signal.SIGUSR1, end_wait_threads)
     34 signal.signal(signal.SIGUSR2, end_wait_threads)
     35 signal.signal(signal.SIGTERM, end_wait_threads)

AttributeError: 'module' object has no attribute 'SIGUSR1'
```

**Comment by michalfita** [comment]
Same for me on Python 2.7.15. I'm not sure if it's Windows specific issue or Python version issue.

**Comment by bprinty** [comment]
This was a Windows-specific issue. Should be fixed in 0.0.6.
